projects/fsastorefrontstyles/scss/overrides/category-page.scss

```diff
@@ -74,18 +74,30 @@ cx-page-layout.FSCategoryPageTemplate {
     }
   }
   .Section3 {
-    cx-fs-cms-custom-container {
+    cx-banner,
+    cx-paragraph {
+      flex: 1 1 50%;
+    }
+    cx-banner {
       @include media-breakpoint-down(sm) {
+        flex: 1 1 100%;
         flex-direction: column;
       }
-      cx-banner,
-      cx-paragraph {
-        flex: 1 1 50%;
+    }
+    cx-banner cx-generic-link,
```

Comment:
Please check if there is need for this.

Reply:
Yes, there is need, because when paragraph text from right side is too long (e.g. German translation), then image is shorter dan paragraph container.